inst/json/graph_prevalence_config.json

```diff
@@ -1,24 +1,28 @@
 {
     "metadata": {
-          "x_col": "month",
-          "y_col": "value",
-          "id_col": "intervention",
-          "format": "long"
+        "x_col": "month",
+        "y_col": "value",
+        "id_col": "intervention",
+        "format": "long"
     },
     "layout": {
-           "title": "Projected prevalence in under 5 year olds",
-           "xaxis": {
-               "title": "years of intervention",
-               "showline": true,
-               "tickvals": [12, 24, 36],
-               "ticktext": [1, 2, 3]
-           },
-           "yaxis": {
-               "title": "prevalence (%)",
-               "showline": true,
-               "range": [0, 1],
-               "autorange": false
-           }
+        "title": "Projected prevalence in under 5 year olds",
+        "xaxis": {
+            "title": "years of intervention",
+            "showline": true,
+            "tickvals": [12, 24, 36],
+            "ticktext": [1, 2, 3]
+        },
+        "yaxis": {
+            "title": "prevalence (%)",
+            "showline": true,
+            "autorange": true,
+            "rangemode": "tozero",
+            "tickformat": ".1%"
+        },
+        "mintcustom": {
+            "hovertemplate": "%{y:.2%}"
```

Comment:
..and we also want to show percentages on the data values - this means adding the hovertemplate value into the dataseries in the front end, but we provide that as a 'mintcustom' here for the front end to apply.

Reply:
I think we'd get this for free, without any changes to the front-end code or having to define any `mintcustom` property, by just defining `hovertemplate` on each series below?

Reply:
Yeah, that makes more sense :facepalm: